go/merge/three_way_set_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package merge
+
+import (
+	"testing"
+
+	"github.com/attic-labs/noms/go/types"
+	"github.com/attic-labs/testify/suite"
+)
+
+func TestThreeWaySetMerge(t *testing.T) {
+	suite.Run(t, &ThreeWaySetMergeSuite{})
+}
+
+type items []interface{}
+
+func (kv items) items() []interface{} {
+	return kv
+}
+
+func (kv items) remove(k interface{}) items {
+	out := make(items, 0, len(kv))
+	for i := 0; i < len(kv); i++ {
+		switch k := k.(type) {
```

Comment:
Small point, but you could save a whole bunch of type switches by determining up-front (i.e. outside the loop) whether `k` is a Value or not. Structure accordingly.